docs/breaking.md

```diff
@@ -0,0 +1,29 @@
+# Breaking changes (migration guide)
+
+This document lists changes introduced in bpftrace which break backwards
+compatibility in some way. Each entry should contain:
+- a link to the PR introducing the change
+- a brief description of the change
+- an example of an error message
+- a simple guide to fix existing scripts
+
+### `pid` and `tid` builtins return `uint32`
+
+https://github.com/bpftrace/bpftrace/pull/3441
```

Comment:
Maybe additional subheads for these to make it super clear e.g.
```
#### PR

#### Error

#### Fix
```

Reply:
Too much visual noise for my liking, I'd prefer not to add it.

Reply:
Yeah, doesn't look like it needs when viewing the rendered markdown.